src/OpenFeature.Contrib.Providers.Flagd/FlagdProvider.cs

```diff
@@ -85,12 +84,6 @@ internal FlagdProvider(Resolver.Resolver resolver)
             _resolver.Init();
         }
 
-        /// <inheritdoc/>
-        public override ProviderStatus GetStatus()
-        {
-            return _status;
-        }
-
```

Comment:
No longer needed (and e2e tests still pass) so everything is working as expected.